### ISSUE TITLE
[Bug]: Generate HL7 fake messages missing sender_id.

### ISSUE THREAD

**Issue by oslynn** [CDCgov/prime-reportstream#9123]
### What happened?

When I generate fake HL7 messages for KS and run it through RS hl7Serializer, it throws exception as below:

Exception in thread "main" gov.cdc.prime.router.ActionError:
Blank value for element (sender_id). Please refer to the ReportStream Programmer's Guide for required fields.
        at gov.cdc.prime.router.serializers.Hl7Serializer.readExternal(Hl7Serializer.kt:438)
        at gov.cdc.prime.router.cli.ProcessData.readReportFromFile(ProcessDataCommands.kt:274)
        at gov.cdc.prime.router.cli.ProcessData.run(ProcessDataCommands.kt:433)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:198)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:211)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:18)
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:400)
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:397)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:415)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:440)
        at gov.cdc.prime.router.cli.MainKt.main(main.kt:296)

To reproduce the problem follow the steps below:
'================================
1.) Generate fake hl7 file:
./prime data --input-fake 5 --input-schema primedatainput/pdi-covid-19 --output-format HL7 --output ./junk/KS/Testing/pdi-covid-19-ks.hl7 --target-states KS

2.) Run through RS prime cli
./prime data --input junk/KS/Testing/pdi-covid-19-ks.hl7 --input-schema HL7/hl7-ingest-covid-19-prod --output-dir junk/KS/output --route

### Version

1.0.2 (Default)

### Relevant log output

```shell
Exception in thread "main" gov.cdc.prime.router.ActionError:
Blank value for element (sender_id). Please refer to the ReportStream Programmer's Guide for required fields.
        at gov.cdc.prime.router.serializers.Hl7Serializer.readExternal(Hl7Serializer.kt:438)
        at gov.cdc.prime.router.cli.ProcessData.readReportFromFile(ProcessDataCommands.kt:274)
        at gov.cdc.prime.router.cli.ProcessData.run(ProcessDataCommands.kt:433)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:198)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:211)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:18)
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:400)
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:397)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:415)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:440)
        at gov.cdc.prime.router.cli.MainKt.main(main.kt:296)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by adegolier** [comment]
@oslynn As far as I can tell there is nothing wrong with the file that's generated here. It passes validation. `sender_id` is not an HL7 field, so it wouldn't be in the file that's generated. I'm a little confused on why you're using a CSV schema to generate a fake HL7 file. Aside from that, the problem appears to be in the --route command. Since this is a command that only works locally, I think I'm going to add in a `--sender` option to let you set a sender_id with the command, unless there's some objection.

**Comment by adegolier** [comment]
After digging into this deeper when I was trying to add the --sender option, I realized that if you have a sender in your local settings that uses the schema you are trying to use when you run the --route command, you don't get that error. I tested this by updating ignore.ignore-hl7 to use the schema `hl7/hl7-ingest-covid-19-prod`, running `./gradlew reloadSettings` and running the second command. I'm closing this issue as not a problem.
